jormungandr/Cargo.toml

```diff
@@ -52,7 +52,7 @@ slog-json = "2.3.0"
 slog-scope = "4.1"
 slog-term = "2.4.0"
 structopt = "^0.3"
-thiserror = "1.0"
+thiserror = "1.0.16"
```

Comment:
Doesn't `1.0` already use the latest path version? Or do we need the specific `0.16` one?

Reply:
This change uses `transparent` in the macro, which was introduced post 1.0 (not a good practice, API additions properly should be associated minor version increments). So I took the latest patch release as the current base version.